python/lsst/ts/MTAirCompressor/aircompressor_csc.py

```diff
@@ -253,7 +258,7 @@ async def end_start(self, data: salobj.type_hints.BaseMsgType) -> None:
             sock = [
                 s for s in self.simulator.server.sockets if s.family == socket.AF_INET
             ][0]
-            self.host, self.port = socket.getnameinfo(sock.getsockname(), 0)  # type: ignore
+            self.host, self.port = socket.getnameinfo(sock.getsockname(), 0)
```

Comment:
Are all consumers of `.port` able to accommodate str named ports?  If not, might be better to keep typing at just `int|None` and validate/convert here?

Reply:
changed to int only